src/spec.c

```diff
@@ -107,13 +115,16 @@ crun_command_spec (struct crun_global_arguments *global_args, int argc, char **a
         libcrun_fail_with_error (errno, "chdir `%s` failed", bundle);
     }
 
-  ret = access ("config.json", F_OK);
-  if (ret == 0)
-    return libcrun_make_error (err, 0, "config.json already exists", err);
+  if (! fname_overriden)
```

Comment:
If you just eliminate this check, you end up with the same code as before, with the ability to override?

Reply:
the check is present because that is what runc does.

I've removed the check when I override the file name so I can do stuff like `crun spec -f /dev/stdout` and print the spec file to stdout so it can be consumed by a pipe without creating a file first.

I am fine to remove the `access` check too though, not sure we care what runc does as anyway `spec` is not an OCI command but an extension